Sprint 10 -JS-Functions/tema3.js

```diff
@@ -0,0 +1,18 @@
+// Scrieți o funcție pentru a găsi numărul de apariții al unei litere într-un șir
+// Funcția letterCount() returnează numărul apariții al unei litere trimise ca parametru dintr-un șir dat.
+
+const letterCount = (str, letter) => {
+    let num = 0;
+    for (let i = 0; i < str.length; i++) {
+        if (str[i].toLowerCase() === letter) {
```

Comment:
e nevoie si la letter s facem toLowerCase, in caz contrar, atunci cand letter e cu majuscula - nu va functiona